setup.py

```diff
@@ -43,6 +43,8 @@
 import warnings
 from datetime import datetime
 
+from pynq.utils import download_overlays
```

Comment:
This line errors out with `RuntimeError: No Devices Found` because it imports `pynq` then `ps`.

Reply:
Understood. However, we need this changes if we want .link files. The only thing I can see doable is to disable this error if no devices are detected. Thoughts?